modules/oe_content_entity/tests/modules/oe_content_entity_test/oe_content_entity_test.info.yml

```diff
@@ -0,0 +1,6 @@
+name: OpenEuropa Corporate Entity Test
+description: Test module for the OpenEuropa Corporate Entities.
+package: Testing
+
+type: module
+core: 8.x
```

Comment:
We need to hide this module or else it will show up in the UI:

![image](https://user-images.githubusercontent.com/153362/75362992-c8d7c780-58b9-11ea-9a0f-4e48bec2c95a.png)

Add:

```yml
hidden: true
```